src/main/java/seedu/address/model/meeting/Meeting.java

```diff
@@ -11,7 +15,7 @@
 public class Meeting {
 
     public static final String MESSAGE_MEETING_CONSTRAINTS =
-            "Meeting should only contain a date and time in DDMMYYHHMM format";
+            "Meeting should only contain a date and time in DD/MM/YY HHMM format";
 
     public static final String MEETING_VALIDATION_REGEX = "\\d{10}";
     public static final String NO_MEETING = "0000000000";
```

Comment:
Maybe in future releases you might want to have a "no meetings scheduled" message displayed, just in case users get confused over this magic 0's